packages/manager/modules/pci/src/projects/new/components/pci-new-project-promotion-banner/index.scss

```diff
@@ -0,0 +1,20 @@
+.pci-new-project-promotion-banner {
+  color: white;
+
+  .promotion-banner {
+    cursor: pointer;
+    background: linear-gradient(180deg, #000e9c, #0050d7) 0% 0% no-repeat
+      padding-box;
+    width: 340px;
```

Comment:
**question:** Is it possible to make it responsive instead of hard-coding the pixels?

Reply:
I had a discussion with the PO regarding this subject. 
The current UI design doesn't alow the banner to be responsible.